region/lowernorfair/east.json

```diff
@@ -3442,25 +4928,46 @@
           "to": [
             {
               "id": 3,
-              "requires": null
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": null
+                }
+              ]
             },
             {
-              "id": 5,
-              "requires": [
-                {"or": [
-                  "canUseSpringBall",
-                  "canUseMorphBombs",
-                  {"obstacle":{
-                    "id": "A",
-                    "requires": [ 
-                      {"enemyKill":{
-                        "enemies": [
-                          [ "Fune" ]
-                        ]
-                      }}
-                     ]
-                  }}
-                ]}
+              "id": 6,
+              "strats": [
+                {
+                  "name": "Spring Ball",
+                  "notable": false,
+                  "requires": ["canUseSpringBall"],
+                  "note": "Use SpringBall to get up and over the Fune."
+                },
+                {
+                  "name": "Morph Bombs",
+                  "notable": false,
+                  "requires": ["canUseMorphBombs"],
+                  "note": "Use a Morph Bomb to get up and over the Fune."
+                },
+                {
+                  "name": "Kill the Fune",
```

Comment:
Wonder if those should go in. You're half a tile higher on the other side, so neither of those strats work on the way back, which means you're risking a softlock.

But I guess it's not for me to decide what rules someone's logic should operate on.

I'll leave the walljump out. I imagine it would cost at least one hit anyway.